src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/EligibilityService.kt

```diff
@@ -19,7 +19,11 @@ fun Collection<EligibilityCheckAndReason>.getIneligibilityReasons(prisoner: Pris
 class EligibilityService(
   private val clock: Clock,
   @param:Value("\${recall.enabled}") private val recallEnabled: Boolean = false,
+  @param:Value("\${recall.prisons}") private val recallEnabledPrisonsEnv: String = "",
+  @param:Value("\${recall.regions}") private val recallEnabledRegionsEnv: String = "",
 ) {
+  val recallEnabledPrisons: List<String> = recallEnabledPrisonsEnv.split(",")
+  val recallEnabledRegions: List<String> = recallEnabledRegionsEnv.split(",")
```

Comment:
You can just read this [directly into a list](https://stackoverflow.com/a/53638956) - don't need to split manually

Reply:
Insane, had no idea this was a thing
Thank you!